source/includes/_events.md

```diff
@@ -1,5 +1,22 @@
 # Events
 
+Every call to action is represented in the API as an event.  Events have related resources such as a contact or a location.
+
+###  Event Attributes
+
+Attribute 	| Data Type | Meaning
+----------  | -------   | -------
+id          | uuid      | unique identifier
+title       | string    | name of the event
+description | string    | breif summary of the event
+free        | boolean   | whether there's a cost for the event
```

Comment:
recommend changing the meaning to "whether the event has no cost to attend", so that the meaning's semantic truthiness matches the attribute's truth value